Sources/XcbeautifyLib/CaptureGroups.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 
 protocol CaptureGroup {
+    static var outputType: OutputType { get }
```

Comment:
What do you think about adding documentation to [`OutputType`](https://github.com/tuist/xcbeautify/blob/a647f8b597983098391fdb57083f2035a357adbf/Sources/XcbeautifyLib/Constants.swift#L21). When looking at this code, it'd help understand what `outputType` represents.